src/main/java/org/bitnick/portfolio/service/ProjectService.java

```diff
@@ -9,6 +32,10 @@
 
 import java.util.List;
 
+/**
+ * This service implementation provides Project entity input/output to
+ * and from the database utilizing the Jpa Repository ProjectRepository.java.
+ */
```

Comment:
Same detail issue as illustrated in UserService.java code review, there is enough detail here.